README.md.in

```diff
@@ -91,6 +91,9 @@ which can have the following values:
   a halt. To recover from this condition, send an explicit ^C to the
   running GDB and continue from the resulting stop.
 
+- `qemu`: Attach via Qemu's GDB server, which is presumed to have the interface available on localhost:1234 
```

Comment:
Isn't the default 3333 in the code?

Reply:
Good catch, I changed it throughout the development process.